Apps/Sandcastle/gallery/development/BillboardClampToGround.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">  <!-- Use Chrome Frame in IE -->
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="Visualize worldwide, high-resolution terrain.">
```

Comment:
This description can be changed manually or with the "info" pulldown inside Sandcastle.